core/src/main/java/io/grpc/CallCredentials.java

```diff
@@ -65,6 +72,7 @@
    * @param applier The outlet of the produced headers. It can be called either before or after this
    *        method returns.
    */
+  @ExperimentalApi("https//github.com/grpc/grpc-java/issues/1914")
   void applyRequestMetadata(
```

Comment:
They don't have to read the documentation, but it does explain what is going on. How can someone implement this API without touching Experimental API? How can someone call this API without touching Experimental API? They can't. But I do agree it is strange, thus the documentation.

> base class that extends this

I don't understand that. I'd have understood "base class than this extends" and "make this a base class that is extended".

> If you want to make this stable and keep experimental methods, please make a base class that extends this. That is the correct place to put methods that may change.

I don't know what you are talking about? That's not a thing we do.

And it doesn't help, since applyRequestMetadata could be completely changed.